source/features/sticky-notifications-actions.tsx

```diff
@@ -0,0 +1,23 @@
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import './sticky-notifications-actions.css';
+import selectHas from '../helpers/select-has';
+
+const FILTER_BAR_SELECTOR = '.js-notifications-container .Box:has(.js-select-all-text)';
+const CLASS_TO_ADD = 'rgh-sticky-notifications-actions';
+
+function init(): void {
+	const filterBox = selectHas(FILTER_BAR_SELECTOR);
+
+	if (filterBox) {
+		filterBox.classList.add(CLASS_TO_ADD);
+	}
+}
+
+void features.add(import.meta.url, {
```

Comment:
Thank you for your PR! 

This feature can be CSS-only, you can drop this entire file. I think the feature looks something like:

```css
.js-notifications-container .Box:has(.js-select-all-text) {
	position: sticky;
	top: 0;
	z-index: 1;
}
```

It's so minor it doesn't need to be disabled. Please ensure it works on small screens too (or at least that it doesn't break anything)

Reply:
> Please ensure it works on small screens too

Also ensure that "select by" and the other dropdown also works correctly, after setting `z-index: 10`

Reply:
I started like that, but the `:has` selector [does not work on firefox for now](https://caniuse.com/css-has), so the `selectHas` js helper is really useful here.

The fact that there is no selector to target this action bar makes me want to limit the side effect too.

Reply:
Unfortunately we're not accepting new code that exists only to work around Firefox bugs. 95% of the users don't need it and maintainers will have to rewrite the feature in the future. It's easier to copy-paste my suggested CSS file than review the current PR’s JS code

Reply:
I should add that Firefox supports some `has` selectors via flag. See:

- #5797

Reply:
OK no problem. I think this will be in firefox soon enough though.

Reply:
Thanks for the screenshots! I think this piece of code should be inside a media query, like `@media (min-height: 500px)`, to ensure we don't cause issues on small viewports